sale_commission/README.rst

```diff
@@ -46,6 +46,8 @@ Contributors
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 * Oihane Crucelaegui <oihanecruce@gmail.com>
 * Nicola Malcontenti <nicola.malcontenti@agilebg.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+  * Carlos Dauden
```

Comment:
Put an empty line above this line for correct RST syntax and include my name here.